crates/cheatcodes/defs/src/vm.rs

```diff
@@ -134,6 +144,49 @@ interface Vm {
         bytes stderr;
     }
 
+    /// The result of a `getStateDiff` call.
+    struct AccountAccess {
+        /// What accessed the account.
+        address accessor;
+        /// The account whose storage was accessed.
+        /// It's either the account created, callee or a selfdestruct recipient for CREATE, CALL or SELFDESTRUCT.
+        address account;
+        /// The kind of account access.
+        AccountAccessKind kind;
+        /// If the account is initialized or empty
+        bool initialized;
```

Comment:
What does `initialized` mean exactly, e.g. what state affects this bool (nonce, code, storage, balance). And this is referring to whether or not it was initialized _before_ the access?

Reply:
This basically indicates whether the account existed just before it was accessed. The account exists if it either has code, a non-zero nonce or a non-zero balance.

Reply:
Clarified.